figures/pipeline/site_daily_metrics.py

```diff
@@ -88,6 +88,32 @@ def get_previous_cumulative_active_user_count(site, date_for):
         return 0
 
 
+def get_site_active_learners_for_date(site, date_for):
+    """
+    Get the active learners ids for the given site and date
+
+    We do this by filtering StudentModule for courses in the site, then
```

Comment:
We don't need to explain code logic here. Remove it

Reply:
This is standard practice in figures API to explain a bit of basic feature of function, however removing it.

Reply:
Done